Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -934,6 +937,10 @@ class AnimatedInterpolation extends AnimatedWithChildren {
     this._interpolation = Interpolation.create(config);
   }
 
+  __makeNative() {
+    super.__makeNative();
```

Comment:
if we are just delegating to `super` here, we should be able to get rid of this whole thing...